autogpt_platform/backend/backend/server/v2/store/db.py

```diff
@@ -492,33 +522,38 @@ async def create_store_submission(
             )
 
         # Create the store listing
-        listing = await prisma.models.StoreListing.prisma().create(
-            data={
-                "agentId": agent_id,
-                "agentVersion": agent_version,
-                "owningUserId": user_id,
-                "createdAt": datetime.now(),
-                "StoreListingVersions": {
-                    "create": {
-                        "agentId": agent_id,
-                        "agentVersion": agent_version,
-                        "slug": slug,
-                        "name": name,
-                        "videoUrl": video_url,
-                        "imageUrls": image_urls,
-                        "description": description,
-                        "categories": categories,
-                        "subHeading": sub_heading,
-                    }
-                },
+        data = prisma.types.StoreListingCreateInput(
+            slug=slug,  # Moved to listing level
+            agentId=agent_id,
+            agentVersion=agent_version,
+            owningUserId=user_id,
+            createdAt=datetime.now(),
```

Comment:
should this become timestamp aware?

Reply:
yess everything has to be timestamp aware, utc